pkg/validate/rsync/validate/source_spec_validator.go

```diff
@@ -199,6 +200,13 @@ func HelmSpec(helm *v1beta1.HelmBase, syncKind string) status.Error {
 		return MissingHelmChart(syncKind)
 	}
 
+	// Validates chart name such that it contains no slashes
+	// See https://helm.sh/docs/chart_best_practices/conventions/ for more details
+	re := regexp.MustCompile("^[^/]*$")
```

Comment:
Can just use `strings.Contains(helm.Chart, "/")`.

I prefer to avoid regex wherever possible, and I think it's overkill here